l10n_ch_payment_slip/payment_slip.py

```diff
@@ -483,9 +483,6 @@ def _draw_address(self, canvas, print_settings, initial_position, font,
         text.moveCursor(0.0, font.size)
 
         address_lines = com_partner.contact_address.split("\n")
-        if com_partner.country_id:
```

Comment:
Some people need to keep contry in adress format for other purposes but country is usually not needed on BVR